components/dark-palette.css

```diff
@@ -49,3 +50,5 @@
 .tribute-container ul li.highlight {
   background: #000;
 }
+
+/*# sourceMappingURL=dark-palette.css.map */
```

Comment:
Oops, this is an artifact from sass CLI command. Excluded the dark-palette.css.map file from the commit.